src/main/java/com/couchbase/lite/replicator/Replication.java

```diff
@@ -641,16 +634,16 @@ public void setFilter(String filterName) {
      * Sets the documents to specify as part of the replication.
      */
     @InterfaceAudience.Public
-    public void setDocIds(List<String> docIds) {
+    public void setDocumentIDs(List<String> docIds) {
```

Comment:
iOS has documentIDs. I will change back to DocIds and let @zgramana review the discrepency.